app/src/main/java/com/project/mobility/view/activities/OnboardingActivity.java

```diff
@@ -63,6 +64,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
         setupViewPager();
         tabNavigation.setupWithViewPager(viewPager);
+        backOnboardingPageButton.setVisibility(View.GONE);
```

Comment:
You can also add the visibility hidden inside the xml and then just add Visibility.VISIBLE where needed